helper.sh

```diff
@@ -224,6 +224,7 @@ function setup_orion_container {
     orion_ctr=${ORION_CTR_PREFIX}"-"${uuid_arg}
     cmd="${DOCKER} run -v ${WS}:${DOCKER_WS} -it -d \
         -w ${DOCKER_WS} \
+        -v /tmp:/tmp \
```

Comment:
For orion we run it within their container. Since we are not using kafka for metrics, it must be taken from /tmp dir. So mounting hosts' /tmp to docker's tmp